core/src/main/java/hudson/model/Computer.java

```diff
@@ -1040,6 +1040,16 @@ public final long getDemandStartMilliseconds() {
     }
 
     /**
+     * Returns the {@link Node} description for this computer
+     */
+    @Exported
+    public @Nonnull String getDescription() {
+        Node node = getNode();
+        return (node != null) ? node.getNodeDescription() : "";
```

Comment:
It is a public API, and without "null" API users may be confused, because empty description is not a undefined description. It is not that important anyway